src/selectors/__tests__/exportContext.ts

```diff
@@ -0,0 +1,78 @@
+import { initialState, reducerFlow } from '../../util'
+import { newThought, setCursor, importText } from '../../reducers'
+import { HOME_PATH } from '../../constants'
+import exportContext from '../exportContext'
+
+describe('exportContext', () => {
+  it('test when meta and archived thoughts are included', () => {
+    const steps = [
+      newThought({ value: 'a' }),
+      newThought({ value: 'b', insertNewSubthought: true }),
+      newThought({ value: 'c' }),
+      setCursor({ path: [{ value: 'a', rank: 0 }] })
+    ]
+
+    // run steps through reducer flow and export as plaintext for readable test
+    const stateNew = reducerFlow(steps)(initialState())
+
+    const exported = exportContext(stateNew, ['a'], 'text/plain', {
+      excludeMeta: false, excludeArchived: false
+    })
```

Comment:
By convention, boolean options can be omitted if they are false. I realize that `false` is more explicit, but we are already omitting them everywhere on the project so I think it's important to be consistent.

Reply:
Done